test/MUXControlsTestApp/Utilities/VisualTreeUtils.cs

```diff
@@ -60,5 +60,21 @@ public static DependencyObject FindVisualChildByName(FrameworkElement parent, st
 
             return null;
         }
+
+        public static T FindElementOfTypeInParentTree<T>(this DependencyObject element)
```

Comment:
I would like to see a consistent naming scheme here used for the methods of this class. I.e.

* FindVisualChildByName -> unchanged
* FindElementOfTypeInSubtree-> FindVisualChildByType
* FindElementOfTypeInParentTree -> FindVisualParentByType

An alternative naming scheme can be found in the Windows Community Toolkit's [VisualTreeExtensions class](https://docs.microsoft.com/en-us/windows/communitytoolkit/extensions/visualtree).

This should probably be done in a separate commit.

(PS: FindVisualChildByName's parent argument should probably be changed to `DependencyObject` as well to create a consistent API surface here.)

Reply:
I think we should rename the other method in this PR, and not introduce a method with a name we currently don't like, to immediately change it in a follow up PR.

Reply:
@chingucoding I would like to do that but for now I went with the existing naming scheme here to be consistent with the current names (mainly FindElementOfTypeInSubtree ). I didn't want to introduce yet another naming scheme here when it is not yet clear how the new names, if we rename them, will look like.

Reply:
Personally I think renaming them should be its own change.  These helpers aren't exposed so I'm not to worried about the name or it changing. Being consistent is nice though. I like your proposal

Reply:
Opened PR https://github.com/microsoft/microsoft-ui-xaml/pull/3438. Once merged, I will merge this PR with master and rename FindElementOfTypeInParentTree() to FindVisualParentByType().